package.json

```diff
@@ -17,7 +17,7 @@
     "deploy": "turbo run deploy",
     "dev": "turbo run dev --scope=\"@calcom/web\"",
     "dev:website": "yarn predev && turbo run dev --scope=\"@calcom/web\" --scope=\"@calcom/website\"",
-    "dev:api": "yarn predev && turbo run dev --scope=\"@calcom/web\" --scope=\"@calcom/api\"",
+    "dev:api": "yarn predev && turbo run dev --scope=\"@calcom/prisma\" --scope=\"@calcom/api\"",
```

Comment:
We don't have a `dev` script on prisma. @agustif

Reply:
Right @zomars, moved it from web to prisma and didn't thought well.

What do you think it's best, to run the whole web before api? We only require the docker/database to be available not really web...
Thoughts?

Reply:
Just remove web if we don't need it

Reply:
done!